### PR TITLE
Adjust stats API to spec.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

**Comment by ricardopereira** [comment]
I run the tests and got:

**ObjC - ARTRestCryptoTest**
```
testSendBinary, ((!error) is true) failed
/Users/ricardopereira/Repositories/Whitesmith/ably-ios/ably-iosTests/ARTRestCryptoTest.m:49
```

```
testSendBinary, (([page count]) equal to (2)) failed: ("1") is not equal to ("2")
/Users/ricardopereira/Repositories/Whitesmith/ably-ios/ably-iosTests/ARTRestCryptoTest.m:56
```

```
testSendBinary, failed: caught "NSRangeException", "*** -[__NSArrayM objectAtIndex:]: index 1 beyond bounds [0 .. 0]"
/Users/ricardopereira/Repositories/Whitesmith/ably-ios/ably-iosTests/ARTRestCryptoTest.m:58
```

Swift tests are passing. :+1:

File: ably-ios/ARTStats.h [hunk 1]
```diff
@@ -9,28 +9,31 @@
...
+
+typedef NS_ENUM(NSUInteger, ARTStatsGranularity) {
+    ARTStatsGranularityMinute,
+    ARTStatsGranularityHour,
+    ARTStatsGranularityDay,
+    ARTStatsGranularityMonth
 };
 
-ART_ASSUME_NONNULL_BEGIN
+__GENERIC(NSArray, NSString *) *ARTStatsIntervalFormatString;
```
Comment (ricardopereira) [inline ably-ios/ARTStats.h]: I don't understand this variable. Why do you need this?

**Comment by ricardopereira** [comment]
LGTM, just a small comment.

File: ably-ios/ARTStats.h (hunk 1 above)
Comment (tcard) [inline ably-ios/ARTStats.h]: You're right, this is a leftover.

**Comment by mattheworiordan** [comment]
LGTM

**Pull request merged**